stable/concourse/templates/web-deployment.yaml

```diff
@@ -233,20 +233,20 @@ spec:
                 secretKeyRef:
                   name: {{ template "concourse.concourse.fullname" . }}
                   key: aws-secretsmanager-secret-key
-            {{- if .Values.secrets.awsSsmSessionToken }}
+            {{- if .Values.secrets.awsSecretsManagerSessionToken }}
```

Comment:
@topherbullock Thanks for that. 

Note that the `awsSsmSessionToken` is a valid secret that is currently being set twice in both the SecretsManager and SystemsManager sections. 

The change made in stable/concourse/templates/web-deployment.yaml line 236 simply corrects the reference to the session token for SecretsManager within the SecretsManager section (rather than `awsSsmSessionToken`). 

The other references you pointed out in values.yaml, secrets.yaml and the README.md files are specific to the SystemsManager (SSM) session token being set in the web-deployment.yaml file line 269 which we probably want to leave as is. These are the values that @Jeeppler is looking to use.

Does that make sense?